### PR TITLE
chore: add more prefixes when editing raw RDF

### PR DESCRIPTION
**Pull request by martinmaillard**

### CODE REVIEW

File: ui/src/components/HydraRawRdfForm.vue [hunk 1]
```diff
@@ -5,6 +5,7 @@
       v-if="shape"
       format="text/turtle"
       :quads.prop="initQuads"
+      :prefixes="editorPrefixes"
```
Comment (tpluscode) [inline ui/src/components/HydraRawRdfForm.vue:8]: Does this vue stringify the array?

File: ui/src/components/HydraRawRdfForm.vue [hunk 2]
```diff
@@ -52,6 +53,7 @@ export default class HydraRawRdfForm extends Vue {
 
   parseError: string | null = null
   editorQuads: Quad[] | null = null
+  editorPrefixes = ['hydra', 'rdf', 'rdfs', 'schema', 'xsd']
```
Comment (tpluscode) [inline ui/src/components/HydraRawRdfForm.vue:56]: FYI `rdf(s)` and `xsd` are not necessary. Always added anyway

**Review by tpluscode — APPROVED** [review]

**Comment by tpluscode** [comment]
On the other hand, I thought we could add a textbook/dropdown where more prefixes could be added as necessary

**Review by l00mi — APPROVED** [review]

**Comment by l00mi** [comment]
@tpluscode lets KISS for now.

File: ui/src/components/HydraRawRdfForm.vue (hunk 1 above)
Comment (martinmaillard) [reply ui/src/components/HydraRawRdfForm.vue:8]: Probably not, why?

File: ui/src/components/HydraRawRdfForm.vue (hunk 2 above)
Comment (martinmaillard) [reply ui/src/components/HydraRawRdfForm.vue:56]: "Explicit is better than implicit"™ 😉

File: ui/src/components/HydraRawRdfForm.vue (hunk 1 above)
Comment (tpluscode) [reply ui/src/components/HydraRawRdfForm.vue:8]: https://zazuko.github.io/rdfjs-elements/rdf-editor/

> `prefixes`: a comma-separated list of prefixes to use for serializing.

Comment (martinmaillard) [reply ui/src/components/HydraRawRdfForm.vue:8]: Funny, it seemed to work 🤔

Comment (tpluscode) [reply ui/src/components/HydraRawRdfForm.vue:8]: Good, so maybe that is what vue does. Native `Array#toString()` simply comma-joins its elements 👌

**Pull request merged**